lib/gettext_i18n_rails/model_attributes_finder.rb

```diff
@@ -65,6 +65,8 @@ def model_attributes(model, ignored_tables, ignored_cols)
 
     def models
       if Rails::VERSION::MAJOR >= 3
+        # Ensure autoloaders are set up before we attempt to eager load!
```

Comment:
could calling setup multiple times be an issue ? <-> should we let this fail and the user deal with ordering ?

Reply:
Subsequent calls to setup after the first are a no-op and are safe.  So this shouldn't be a problem.